react/components/Tooltip/tooltip.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { withKnobs, text, boolean, select, number } from "@storybook/addon-knobs"
+
+import Tooltip from '.'
+import Button from '../Button'
+
+export default {
+  title: 'Components|Tooltip',
+  component: Tooltip,
+  decorators: [withKnobs]
+}
+
+const positions = ['top', 'left', 'right', 'bottom']
+
+export const Default = () => (
+  <Tooltip
+    label={text('Label', 'Tooltip Label')}
+    trigger={select('Trigger', ['click', 'hover', 'focus'], 'hover')}
+    wordBreak={select('Word Break', ['normal', 'break-all', 'keep-all', 'break-word'], 'normal')}
+    position={select('Position', positions, 'left')}
+    delay={number('Delay(ms)', 0)}
+    fallbackPosition={select('Fallback Position', positions, 'right')}
+  >
+    <Button>Button</Button>
+  </Tooltip>
+)
+
+export const AlwaysVisible = () => (
+  <Tooltip label="Tooltip Label" trigger="focus">
+    <Button autoFocus={true}>Button with focus</Button>
+  </Tooltip>
+)
```

Comment:
this example does not make sense since this is not exactly a tooltip feature, it sounds more like a hack to make the tooltip always active ... but it only works with a button and the tooltip is not exclusive for use only with a button.
```suggestion
```

Reply:
I added this example to help people test the tooltip without having to hover the tooltip children but I agree that doesn't make sense :sweat_smile: I'll remove it